.github/workflows/rust.yml

```diff
@@ -226,6 +226,37 @@ jobs:
           cd rust
           cargo clippy --all-targets --workspace -- -D warnings -A clippy::redundant_field_names
 
+  miri-checks:
+    name: Miri Checks
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64]
+        rust: [stable]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-miri-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install
+        uses: actions-rs/toolchain@v1
```

Comment:
This action is blocked by infra. Could you try using one of the commands on the other steps?

Reply:
Cool, I have changed it. Now it has started.